packages/ssr-web/app/services/app-context.ts

```diff
@@ -0,0 +1,28 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
+import Fastboot from 'ember-cli-fastboot/services/fastboot';
+
+export interface AppContextService {
+  currentApp: 'card-space' | 'wallet';
+  cardSpaceId: string;
+}
+
+export default class AppContext extends Service implements AppContextService {
+  @service declare fastboot: Fastboot;
+
+  get currentApp(): 'card-space' | 'wallet' {
+    // return 'wallet';
+    return 'card-space';
+  }
+
+  get cardSpaceId() {
+    return 'nosuchmerchant';
+  }
+}
```

Comment:
Intention is for this to centralize the knowledge of which app we are serving, and if it is card space, provide the id.

Reply:
This should also allow us to hide the local-dev specific hacks.

Reply:
I've figured out how to get local DNS hax in so we don't have to use a different way to get the id for dev, will post the method here when I've ironed out the details

Reply:
@backspace